content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,218 @@
+:guilabel:`Country` in it. Type :guilabel:`Date` in the :guilabel:`Search...` bar, or scroll to
+search through the list to locate it. Then, click the next field and select :guilabel:`>=` from the
+drop-down menu. This operator will include only values greater than or equal to the value in the
+rightmost field.
+
+The rightmost field should contain the earliest date leads are selected from. For example,
+setting `01/01/2024 00:00:05` will include only leads created from, and including, the first day of
+2024.
+
+.. image:: lead_distribution_report/created-on.png
+   :align: center
+   :alt: Add a Created On rule for the start of the year onward.
+
+Add sales team location
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule and another horizontal set of fields will appear. Click
+the first field for the new rule, and set it to :guilabel:`Sales Team`. Then, click the next
+field and select :guilabel:`contains` from the drop-down menu. Selecting this operator will filter
+for any records that contain the words in the rightmost field.
+
+In this third field, enter the name of the desired sales teams. For example, setting `us direct sf
+northam` will include only sales teams assigned to the U.S., direct, San Francisco, and North
+America.
+
+.. image:: lead_distribution_report/sales-team-location.png
+   :align: center
+   :alt: Use Sales Team to filter the location the lead is associated with.
+
+Add phone number
+~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, click the next field and select :guilabel:`is set` from the drop-down menu.
+Selecting this operator will filter only for records that have a phone number associated with the
+lead.
+
+.. image:: lead_distribution_report/phone-set.png
+   :align: center
+   :alt: Report only leads with an associated phone number with the Phone field.
+
+Add active status
+~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of :guilabel:`Phone is set` to add a new rule that
+branches from the rules above.
+
+Two horizontal sets of fields will appear below a line showing
+:guilabel:`all of:` with a :icon:`fa-caret-down` down arrow. Click the down arrow, then select
+:guilabel:`any` from the list.
+
+Set the first field to :guilabel:`Active`. Then, click the rightmost field and select
+:guilabel:`is set` from the drop-down menu.
+
+.. image:: lead_distribution_report/add-branch.png
+   :align: center
+   :alt: Click the second button to the right of phone is set labeled add branch.
+
+Next, click the :icon:`fa-plus` :guilabel:`New Rule` button next to :guilabel:`Active is set` to
+create a new set of fields. Set the first field to :guilabel:`Active`. Then, click the next field
+and select :guilabel:`is not set` from the drop-down menu. This rule will add the activity status of
+the lead to the report.
```

Comment:
```suggestion
Next, click the :icon:`fa-plus` :guilabel:`New Rule` button next to :guilabel:`Active is set` to
create a new line of fields beneath it.

Set the first field to :guilabel:`Active`. Then, click the next field and select :guilabel:`is not set` from the drop-down menu. This rule will add the activity status of the lead to the report.
```
This second part of this needs to be rewritten. When I click the New Rule button next to active is set, a new rule line appears, that already duplicates the entries from the line above it -- meaning, with those fields mostly filled in, all I'd have to is change the third field to 'not set' to accomplish this..

please look into this, and revise, as needed. if you have any questions, don't hesitate to let me know.

Reply:
Also, when we clicked Branch earlier, that reveals 2 new lines, one of which we altered (Active), but the other is still on the default 'Country' option, is that by design? Might be beneficial to use that line or mention why its being overlooked, so as to avoid any confusion with the reader.